lib/src/emoji_picker.dart

```diff
@@ -231,12 +213,7 @@ class EmojiPickerState extends State<EmojiPicker> {
         _emojiPickerInternalUtils
             .addEmojiToRecentlyUsed(emoji: emoji, config: widget.config)
             .then((newRecentEmoji) => {
-                  _recentEmoji = newRecentEmoji,
-                  if (category != Category.RECENT && mounted)
-                    setState(() {
-                      // rebuild to update recent emoji tab
-                      // when it is not current tab
-                    })
```

Comment:
This is needed to avoid jumping of emoji's inside Recent-Tab. I would consider it bad UX when every time when you select a emoji from the recent-tab jumps around because the use-count increases.

Reply:
Oh, I missed the point. Will fix and add a comment about reasoning for this check.